nixos/modules/security/hidepid.nix

```diff
@@ -33,6 +33,12 @@ in
   };
 
   config = mkIf cfg.enable {
+    assertions =
+     [ { assertion = cfg.procGroup != null -> (config.users.groups."${cfg.procGroup}" or "") != "";
+         message = "the group named at security.hideProcessInformation.procGroup does not exist!";
+       }
+     ];
+
```

Comment:
It would also have the benefit of working even if group info was not available for some reason (I *think* that'll never be the case, but still). I will switch to a static group id, then.